testsuite/MDAnalysisTests/converters/test_rdkit.py

```diff
@@ -255,9 +255,9 @@ def test_error_no_hydrogen(self, uo2):
             uo2.atoms.convert_to("RDKIT")
 
     def test_error_no_hydrogen_implicit(self, uo2):
-        with pytest.warns(None) as record:
+        with warnings.catch_warnings():
+            warnings.simplefilter("error")
             uo2.atoms.convert_to.rdkit(NoImplicit=False)
-        assert len(record) == 0
```

Comment:
It's a bit odd not to assert anything here but I also don't know what should be asserted if we just try to get through the code path.

Reply:
So here I see it as the simplefilter is a type of assert (i.e. it asserts that there are no warnings)